routing/pathfind_test.go

```diff
@@ -314,6 +314,7 @@ type testChannelPolicy struct {
 	FeeRate     lnwire.MilliSatoshi
 	LastUpdate  time.Time
 	Disabled    bool
+	Initiator   bool
```

Comment:
I think this is better named `Direction`, since it doesn't reflect who initiated the channel

Reply:
Agree. Changed to use Direction.